db/migrate/20230210103726_add_users_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :users, null: false, foreign_key: true
+  end
```

Comment:
- [ ] According to the [RD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) a `Comment` should belong to a `:author` with foreign_key in the` users` table, I suggest you update this to something like:
```ruby
add_reference :comments, :author, foreign_key: { to_table: 'users' }`
```